Sources/Web3Core/Utility/Array+Extension.swift

```diff
@@ -106,3 +106,12 @@ extension Array where Element: BinaryInteger {
         return sorted_data[index]
     }
 }
+
+// MARK: - Conversion
+
+/// Transforms `[Any?]` into `[AnyObject]`
+extension Array where Element == Any? {
```

Comment:
1) This extension doesn't apply to `[Any]` but it should. As a solution `where` clause can be removed;
2) I'd suggest renaming the function to `toAnyObjectArray`;
3) Why `self.map { ... }` and not just `as [AnyObject]`?

This is the suggested update:
```swift

/// Transforms into `[AnyObject]`
extension Array {
    func toAnyObjectArray() -> [AnyObject] {
        self as [AnyObject]
    }
}
```